app/models/api_namespace.rb

```diff
@@ -97,7 +97,16 @@ class ApiNamespace < ApplicationRecord
     else
       self.none
     end
-   }
+  }
+
```

Comment:
@Pralish is this change necessary?

Reply:
@donrestarone , it's not necessary. The indentation was one space ahead, so fixed it.